bigquery_etl/static.py

```diff
@@ -118,13 +110,6 @@ def _load_table(
 
     job.result()
 
-    if description_file_path is not None:
-        with open(description_file_path) as description_file:
-            description = description_file.read()
-            table = client.get_table(table_ref)
-            table.description = description
-            client.update_table(table, ["description"])
```

Comment:
IMO it'd be nice if `bqetl static publish` still wrote the table's description/metadata, just based on a `metadata.yaml` file if one exists instead of `description.txt` (e.g. it could import and call the `publish()` function from `bigquery_etl.cli.metadata`).

As it is, it seems like we'd be relying on the `publish_metadata` task in the `bqetl_artifact_deployment` DAG running to publish the metadata for such static tables, but that task is downstream of the `publish_new_tables` and `publish_views` tasks which have had a tendency to break.

Reply:
In the linked pr https://github.com/mozilla/telemetry-airflow/pull/1934, publishing of static tables is being moved downstream of metadata publishing so we would need to not do that.  Although this does make me think that they should be kept separate with `bqetl static publish` still writing the description since it will create a new table anyway and doesn't need to depend on artifact deployment.

But I don't feel strongly either way since table or view deployment failure happening the same day as a static table change is probably rare and we should focus on fixing the deployment in that case.

Reply:
> table or view deployment failure happening the same day as a static table change is probably rare

It's not that rare.  I've had several PRs that both added a new static table and used it in downstream ETLs/views (usually when I add one it isn't for a standalone purpose).